1.1.0-msbuild/init_container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Ah you're right, forgot that was at the root and covered all matching filenames in the repo.